src/lib/sort/sort-header.ts

```diff
@@ -83,15 +84,16 @@ export class MdSortHeader implements MdSortable {
   get _id() { return this.id; }
   set _id(v: string) { this.id = v; }
 
-  constructor(public _intl: MdSortHeaderIntl,
+  constructor(public intl: MdSortHeaderIntl,
```

Comment:
Do you mean for this to be part of the public api now?

Reply:
Trying to find the discussion about it, but can't find it. It seems people already are using it and would like to have it part of the public API. Wrong PR to put this in, but curious what you think about allow people to access it directly